livekit-android-sdk/src/main/java/io/livekit/android/e2ee/E2EEManager.kt

```diff
@@ -37,14 +37,18 @@ class E2EEManager
 constructor(keyProvider: KeyProvider) {
     private var room: Room? = null
     private var keyProvider: KeyProvider
-    private var frameCryptors = mutableMapOf<String, FrameCryptor>()
+    private var frameCryptors = mutableMapOf<Map<String, String>, FrameCryptor>()
```

Comment:
```suggestion
    private var frameCryptors = mutableMapOf<Pair<String, String>, FrameCryptor>()
```

Looks like the key of frameCryptors should just be an immutable pair rather than a full map?